packages/framework/bundle/config/values.yaml

```diff
@@ -42,4 +42,4 @@ tanzuAuthPackageValues:
   namespace: tanzu-auth
   versionConstraints:
   controller:
-    image: ""
+    image: tanzu-auth-controller-manager:latest
```

Comment:
Please do add the remaining fields as per the `values.yaml` in the `tanzu-auth` package
```
tanzuAuthPackageValues:
  createNamespace: true
  namespace: tanzu-auth
  versionConstraints:
  controller:
    image: ""
    image: tanzu-auth-controller-manager:latest
  deployment:
    hostNetwork: true
    nodeSelector: null
    tolerations: []
```

Reply:
Ok all of the vals are in there with the defaults.